WordPress/Docs/WP/EnqueuedResourcesStandard.xml

```diff
@@ -0,0 +1,50 @@
+<documentation title="Enqueued Resources">
+    <standard>
+    <![CDATA[
+    Scripts must be registered/enqueued via wp_enqueue_script().
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Script registered and enqueued correctly.">
+        <![CDATA[
+<em>wp_enqueue_script</em>( 
+    'someScript-js', 
+    'http://someurl/somefile.js', 
+    array( 'jquery' ),
+    '1.0.0',
+    true
+);
+        ]]>
+        </code>
+        <code title="Invalid: Script is directly embedded in HTML.">
+        <![CDATA[
+<script <em>src="http://someurl/somefile.js"</em>>
+</script>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Stylesheets must be registered/enqueued via wp_enqueue_style.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Stylesheet registered and enqueued correctly.">
+        <![CDATA[
+<em>wp_enqueue_style</em>( 
+    'style-name', 
+    'https://example.com/somefile.css', 
+    false, 
```

Comment:
The third parameter of `wp_enqueue_style()` is expected to be an array.

Ref: https://developer.wordpress.org/reference/functions/wp_enqueue_style/